Engines/Wine/Settings/always offscreen/script.js

```diff
@@ -4,45 +4,50 @@ include("engines.wine.plugins.regedit");
 /**
  * setting to set always offscreen
 */
-var settingImplementation = {
-    _options: [tr("Default"), tr("Disabled"), tr("Enabled")],
-    // values which are written into the registry, do not translate!
-    _registryValues: ["", "disabled", "enabled"],
-    getText: function () {
+class AlwaysOffscreenSetting {
+    constructor() {
+        this.options = [tr("Default"), tr("Disabled"), tr("Enabled")];
+        // values which are written into the registry, do not translate!
+        this.registryValues = ["", "disabled", "enabled"];
+    }
+
+    getText() {
```

Comment:
I think it would be a good idea to change this method to a JavaScript getter method:

```javascript
get text() {
   return tr("Always offscreen");
}
```

What do you think?

Reply:
I'm not that familiar with Javascript getters. Any advantages (except that it looks nicer). Can it still be called the same way from Java?

Reply:
> I'm not that familiar with Javascript getters. Any advantages (except that it looks nicer).

No, this is only a cosmetical change.

> Can it still be called the same way from Java?

I still need to check. I just want to discuss this as early as possible

Reply:
I suggest to implement the simplification first without this and then have a look at it again.

Reply:
I've just tried if `get text()` can be used as a synonym for `getText` in the Java interface, but it doesn't seem to work. I've opened an [issue](https://github.com/graalvm/graaljs/issues/181) about this in the GraalJS GitHub repository to see whether this is intentional or not.